### PR TITLE
SSE-2643: Enable AWS WAF for frontend load balancer

### PR DESCRIPTION
**Pull request by abdul-gds**
# Code Changes

- add WAF resources for adding security
- AWS-CRS
- Bad-Inputs
- SQLInject-RuleSet
- RateBased-CountIpRule
-web Acl logging

### CODE REVIEW

File: infrastructure/ci/dev-platform/frontend-template.yml [hunk 1]
```diff
@@ -339,6 +339,196 @@ Resources:
         Fn::ImportValue:
           !Sub ${VpcStackName}-VpcId
 
+
+  frontendWebAcl:
```
Comment (CharlesIC) [inline infrastructure/ci/dev-platform/frontend-template.yml:343]:
```suggestion
  FrontendWebAcl:
```
And the same for the other new resources to use the same naming convention as we use in the rest of the template

**Review by CharlesIC — COMMENTED** [review]
Small issue with naming resources and the conflict needs to be resolved, otherwise the PR looks good

File: infrastructure/ci/dev-platform/frontend-template.yml (hunk 1 above)
Comment (abdul-gds) [reply infrastructure/ci/dev-platform/frontend-template.yml]: fixed

**Comment by abdul-gds** [comment]
closing now, might we need to use it later

**Pull request closed without merging**